lib/util/fileUtil.js

```diff
@@ -76,6 +86,61 @@ function listDirectories(dirPath) {
 }
 
 /**
+ * Open the given package.json file path and read the given properties
+ * into an object.
+ *
+ * @param {string} filePath path to a package.json file.
+ * @param {string[]} properties the properties that should be extracted.
+ * @returns {Object} an object with the given properties from package.json.
+ */
+function readPackageJson(filePath, properties) {
```

Comment:
Why the need to give in a list of properties, can't the function just return whatever is in the package.json object?